herd/ASLAction.ml

```diff
@@ -10,21 +10,34 @@ module Make (A : S) = struct
   module A = A
   module V = A.V
 
+  type annot = AcqSc|AcqPc|RelSc|Std
```

Comment:
Can you add a comment that explain those symbols, and say what they refer to? Because at the moment I don't understand what is written here.